src/theme.ts

```diff
@@ -13,22 +13,43 @@ export const theme = {
     WHITE: "#FFFFFF",
   },
   hanehldaColors: {
-    CREAM: "#F5EBD5",
-    TEXT_CREAM: "#FFEA9F",
+    CREAM: "#fbf8f0",
     DARK_YELLOW: "#F7C85D",
-    DARK_BLUE: "#489ABD",
+    DARK_BLUE: "#6a9eb5",
     ERROR_RED: "#FFCED0",
-    LIGHT_RED: "#DFBFAD",
-    DARK_RED: "#98342E",
+    LIGHT_RED: "#c08176",
+    DARK_RED: "#a14738",
     BORDER_GRAY: "#D9D9D9",
-    DARK_GRAY: "#464D50",
+    DARK_GRAY: "#1a2327",
     LIGHT_GRAY: "#707070",
     TEXT_LIGHT_GRAY: "#898989",
-    LIGHT_GREEN: "#62B223",
+    LIGHT_GREEN: "#285c00",
     DARK_GREEN: "#0B455E",
+    
+    HEADERBAR: "#a14738",
+    WHITETEXT: "#ffffff",
+    DARK_BLUE_TEXT: "#0c455e",
+    DARK_RED_TEXT: "#390c05",
+    SIDEBAR: "#a94f3f",
+    LIGHT_YELLOW: "#f7ecc7",
+    TABLE1: "#f9f4e4",
+    TABLE2: "#fdfbf5",
+    WHITE_BUTTON: "#faf3f3",
+    WHITE_HIGHLIGHT: "#f4dba6",
+    GRAY_HIGHLIGHT: "#52504e",
+    DARK_RED_HIGHLIGHT: "#72190a",
+    ERROR_RED_DARKER: "#e89b9e",
+    DARK_GREEN_TEXT: "#00200b",
+    CORRECT_GREEN: "#c0fccb",
+    YELLOW_BUTTON: "#f6eebc",
+    YELLOW_HIGHLIGHT: "#eade96",
   },
   borderRadii: {
-    md: "20px",
+    md: "7px",
+    md_small: "4px",
+    b: "none",
+    bs: "rgba(0, 0, 0, 0.15) 2.4px 2.4px 3.2px",
+    bb: "4px solid rgba(0, 0, 0, 0.1)",
   },
```

Comment:
Three things, mostly about naming. In general, these are a little terse to me, and the letters are free.
1. I don't like having `b: "none"` I don't know what `b` means in downstream code, but I do know what `"none"` means.
2. Why is there `md_small` but no `small` could this just be `sm` or `small`?
3. `bs` should go on a separate `boxShadows: { md: "..." }` or `default` maybe.
4. I have no idea what `bb` means. In material design, I think this style of border is called a "seam" so we could use that?